source/rst/lss_examples.rst

```diff
@@ -40,9 +40,9 @@ Assume that
 
 -  :math:`\{\epsilon_{t}\}` is a sequence of independently and
    identically distributed :math:`N(0,1)` random variables, i.e., mean
-   zero, variance one.
+   zero, variance one
 
--  $:raw-latex:`\sigma `:raw-latex:`\geq `0 $ is a "volatility"
+-  :math:`\sigma` :math:`\geq` 0 is a "volatility"
```

Comment:
This should be :math:`\sigma \geq 0`

Reply:
This still needs fixing

Reply:
Actually this is in another file -- can @AnjuJoon figure out how to reverse changes to this file so we don't merge in any changes here?